xmtp_mls/src/groups/message_history.rs

```diff
@@ -42,19 +43,23 @@ use crate::{
 
 const ENC_KEY_SIZE: usize = 32; // 256-bit key
 const NONCE_SIZE: usize = 12; // 96-bit nonce
+const HISTORY_SERVER_HOST: &str = "0.0.0.0";
```

Comment:
should host/server be only in `mod test`?

Reply:
Yep. I've moved them to `mod tests`  .  I haven't finalized on how/where integrators should declare the `host:port` for their own message-history-server.  Likely an `std::env::var` that defaults back to ours if not provided?
Another thing to discuss and roll into a follow up PR.